packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -149,6 +149,11 @@ const propTypes = {
    * With this property the height of the cell will grow to fit the cell content.
    */
   rowMinimumHeight: PropTypes.string,
+
+  /**
+   * Determines if focus is moved to the interactive element of a cell when a single button or hyperlink element is the only interactive element.
+   */
+  isAutoFocusEnabled: PropTypes.bool,
```

Comment:
would `enableAutofocus` make sense as a simpler name?

Reply:
I tried to follow the naming pattern of some of the other props.  However, I can make this change if it is the desired update.

Reply:
I think the `is___` pattern is preferred. Would a name like `isAutofocusable` or similar work? (Not sure if that's sufficiently descriptive of the prop's usage though)

Reply:
If we're keeping to the `is__` pattern, then I think both these names are similar enough that either works.